pkg/steps/clusterinstall/template.go

```diff
@@ -442,7 +442,8 @@ objects:
           mkdir -p /tmp/artifacts/pods /tmp/artifacts/nodes /tmp/artifacts/metrics /tmp/artifacts/bootstrap /tmp/artifacts/network
 
 
-          if [ -f /tmp/artifacts/installer/terraform.tfstate ]
+          # NOTE: OpenShift 4.2 and greater have native 'gather bootstrap' support
+          if [ -f /tmp/artifacts/installer/terraform.tfstate ] && openshift-install version | grep v4.1 &>/dev/null
```

Comment:
The CI runs have 0.0.1- style versions. i'm not sure if CI jobs running on release-4.1 branches will have the 4.1XX version set??

Reply:
I got it from one of these:

https://storage.googleapis.com/origin-ci-test/logs/release-openshift-ocp-installer-e2e-aws-4.1/347/artifacts/e2e-aws/installer/.openshift_install.log

Which has: "OpenShift Installer v4.1.12-201908130938-dirty"

Are there other branches to account for?